src/ray/raylet/worker_pool.h

```diff
@@ -455,8 +459,11 @@ class WorkerPool : public WorkerPoolInterface, public IOWorkerPoolInterface {
     /// A map for looking up the task with dynamic options by the pid of
     /// worker. Note that this is used for the dedicated worker processes.
     std::unordered_map<Process, TaskID> dedicated_workers_to_tasks;
-    /// A map for speeding up looking up the pending worker for the given task.
-    std::unordered_map<TaskID, Process> tasks_to_dedicated_workers;
+    /// A set for speeding up looking up the pending worker for the given task.
+    std::unordered_set<TaskID> tasks_to_dedicated_workers;
```

Comment:
`tasks_with_dedicated_workers`. The comment needs to be changed as well. It can be something like "All tasks that have associated dedicated workers.".